ckan/controllers/user.py

```diff
@@ -162,10 +166,90 @@ def register(self, data=None, errors=None, error_summary=None):
 
         return self.new(data, errors, error_summary)
 
+    def _generate_internal_logs(self, log_message):
+        '''
+        This helper function is called if sending a new user registration email fails.
+        '''
+
+        logging.warning("There was an error sending the email. Writing to log file.")
+
+        log_path = os.environ['REGISTRATION_ERROR_LOG_FILE_PATH']
```

Comment:
best to check that this actually exists in the environment or this will die with an exception

Reply:
Fixed.